runtime/v2/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	cosmossdk.io/server/v2/stf v0.0.0-00010101000000-000000000000
 	cosmossdk.io/store/v2 v2.0.0-00010101000000-000000000000
 	cosmossdk.io/x/tx v0.13.3
+	github.com/cometbft/cometbft v0.38.10
```

Comment:
this looks like something we want to avoid, can it be removed

Reply:
had a non committed file that i was using for testing that causes  this